addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -761,6 +762,9 @@ function makeActionManager(env) {
                 });
             }
         }
+        if (options.onClose) {
```

Comment:
I'd do that in the else to reflect what was done in the ActionManager

Reply:
Ok, I'll change that